src/ModuleInstantiation.h

```diff
@@ -42,7 +43,8 @@ class IfElseModuleInstantiation : public ModuleInstantiation {
 	IfElseModuleInstantiation(shared_ptr<class Expression> expr, const std::string &source_path, const Location &loc) : ModuleInstantiation("if", AssignmentList{Assignment("", expr)}, source_path, loc) { }
 	~IfElseModuleInstantiation();
 	std::vector<AbstractNode*> instantiateElseChildren(const Context *evalctx) const;
-	void print(std::ostream &stream, const std::string &indent) const override;
+	void print(std::ostream &stream, const std::string &indent, const bool inlined) const final;
+	void print(std::ostream &stream, const std::string &indent) const final { print(stream, indent, false); };
```

Comment:
What's the purpose of this? It seems to have the same implementation as its supertype, except it's `final`?

Reply:
Yeah looks like its unnecessary, removed now.